tests/python/apps/solana_utils.py

```diff
@@ -52,3 +56,81 @@ def lamports_to_bytes(lamports: int) -> str:
 ### Package this currency configuration in exchange format ###
 
 SOL_CONF = create_currency_config("SOL", "Solana")
+
+
+def enable_blind_signing(navigator, device_name: str, snapshots_name: str):
+    if device_name.startswith("nano"):
+        nav = [NavInsID.RIGHT_CLICK, NavInsID.BOTH_CLICK, # Go to settings
+               NavInsID.BOTH_CLICK, # Select blind signing
+               NavInsID.RIGHT_CLICK, NavInsID.BOTH_CLICK, # Enable
+               NavInsID.RIGHT_CLICK, NavInsID.RIGHT_CLICK, NavInsID.RIGHT_CLICK, NavInsID.BOTH_CLICK # Back to main menu
+              ]
+    else:
+        nav = [NavInsID.USE_CASE_HOME_SETTINGS,
+           NavIns(NavInsID.TOUCH, (348,132)),
+           NavInsID.USE_CASE_SETTINGS_MULTI_PAGE_EXIT]
+    navigator.navigate_and_compare(ROOT_SCREENSHOT_PATH,
+                                   snapshots_name,
+                                   nav,
+                                   screen_change_before_first_instruction=False)
```

Comment:
There is a bit of Stax related code here, but don't bother changing it, It's just forward test implementation following the concept of creating test before coding the functionality!